### ISSUE TITLE
minor issue with youtube

### ISSUE THREAD

**Issue by japharl** [browsh-org/browsh#440]
videos no longer play for some reason in new version, on raspbian.  Poor rick doesn't roll.

**Comment by tombh** [comment]
Yes I've noticed that too. I haven't figured out why yet. I think videos can play in theory, it's just that Youtube isn't recognising the click to start playing. Let's call this a bug then

**Comment by japharl** [comment]
Yup, no rush to fix but an issue.

**Comment by guoquan** [comment]
With GUI on, it seems video is blocked defaultly by FF

![image](https://github.com/user-attachments/assets/be39b69e-0545-4f05-9f4e-cd2145ef5cbf)

**Comment by guoquan** [comment]
I am not super familiar with Firefox preferences (`about:config`). It seems the autoplay setup setting is per site. I have checked that, when I have YouTube set to `Allow Audio and Video` and then visit Vimeo, it is still blocked by default. The setting goes to Firefire - Settings - Browser Privacy - Permissions - Autoplay. And there is a list of websites and setting status.

**Comment by guoquan** [comment]
Trying to toggle the "Default for all websites" and filter the items on `about:config`.
It is corresponds to key `media.autoplay.default` where the value
- `0` for "Allow Audio and Video",
- `1` for "Block Audio" (Firefox default),
- and `5` for "Block Audio and Video".
It seems even "Block Audio" (`1`) would prevent these websites from playing.

Want to give it (`0`) a try?

**Comment by tombh** [comment]
Thanks, so I added this to my `~/.config/browsh/config.toml`:

```
preferences = [
	"media.autoplay.default=0"
]
```

But still Youtube videos won't play, neither automatically on page load, nor when I click anywhere on the video. So I'm lost again as to what the problem is 😞

**Comment by guoquan** [comment]
So, I am on macOS, browsh is installed with homebrew, and has a different `config.toml` path `~/Library/Application Support/browsh/config.toml`.
I did `browsh --debug` and checked `./debug.log` according to https://www.brow.sh/docs/config/ to find it.

I think adding this one does the job for me.

> ```toml
> preferences = [
>   "media.autoplay.default=0"
> ]
> ```

**Comment by tombh** [comment]
Oh! For you it made videos play!?

**Comment by guoquan** [comment]
Yes, it did! I cleared my per-site permission list, and both YouTube and Vimeo are still playing.

You may want to make sure that 
- the `config.toml` preference takes effect in `about:config`,
- and then `media.autoplay.default=0` correspond to "Allow Audio and Video"
- and then "Allow Audio and Video" enables playing video.

**Comment by tombh** [comment]
So you're saying that manually changing `media.autoplay.default` in Firefox did it for you? Or that you put `media.autoplay.default=0` in Browsh's config and that did it for you?

**Comment by guoquan** [comment]
Either way works for me.

I started with manually changing `media.autoplay.default` to `0` in `about:config`. It worked.
And I reset it to default (`1`). It stopped playing then.
I found my `config.toml`, and added `media.autoplay.default=0` to `preference`. Now it is working again.

So, I am assuming this is working.
